docs/UserGuide.md

```diff
@@ -371,6 +365,15 @@ Optional fields for a **recurring** lesson is:
 * End date
 * Cancelled dates
 
+TAB has features to help to prevent scheduling mistakes.
+For example, we help you avoid mistakes scheduling two lessons at the same slots by ensuring that lessons never overlap.
+Additionally, we help to avoid accidentally scheduling lessons outside working hours.
+
+In the future, we intend to add some nice-to-have features that will allow you to customise your working hours,
+as well as specify minimum/maximum lesson durations so that accidents like mistyping a 10-minute lesson won't happen.
+Additionally, we will allow you to toggle these checks on and off, just in case you don't want them.
+Do look forward to these features!
```

Comment:
I don't quite understand these additional future features and is there a need to mention these? >.<

Reply:
I think these are to mitigate the feature flaws claims as upcoming features